tools/dockerfile/grpc_iwyu/iwyu.sh

```diff
@@ -30,7 +30,7 @@ cat compile_commands.json | sed "s,\"file\": \",\"file\": \"${IWYU_ROOT}/,g" > c
 
 # figure out which files to include
 cat compile_commands.json | jq -r '.[].file' \
-  | grep -E "^src/core/lib/(promise|uri)/" \
+  | grep -E "^src/core/lib/(promise|resource_quota|uri)/" \
```

Comment:
is this meant to be modified?

Reply:
Yeah... it's the 'opt-in' mechanism for turning on this tool on a directory... eventually it'll collapse to `src/core/` or some such thing, but for now it's getting incrementally expanded.

Reply:
Ah ok.